### PR TITLE
bring table styles closer to the spec

### PR DESCRIPTION
**Pull request by TitanNano**
We changed the styles to be closer to the material spec.

### CODE REVIEW

**Comment by TitanNano** [comment]
@fredguile would you agree to merge this change? otherwise we need to publish our own package.

**Comment by fredguile** [comment]
Hi Jovan, thanks for your contribution! I'd be ok to merge it but I'm currently unable to reach any computer within 2 weeks :( Would you be ok to  temporarily use your package during this time until I publish an update? I appreciate that you make this datatable even stronger. I can review your PR from my phone in the meantime.

File: src/_md-datatable-theme.scss [hunk 1]
```diff
@@ -72,33 +62,44 @@
...
       width: $check-cell-width;
     }
   }
 
   ng2-md-datatable-column {
-    padding: 0 $datatable-column-spacing 0 $datatable-column-padding;
+    padding: 0;
     width: $cell-width;
-    color: md-color($foreground, secondary-text);
+    color: #5c5c5c;
```
Comment (fredguile) [inline src/_md-datatable-theme.scss:102]: Do you think we could leave the colors obtained from $foreground, $background and $accent so as to keep theming possibilities of Material 2?

File: src/md-datatable-row.component.scss [hunk 2]
```diff
@@ -1,13 +1,43 @@
+$first-cell-padding: 24px;
+$cell-padding: 56px;
+
 :host {
+  $hover-color: #EEEEEE;
+  $selected-color: #F5F5F5;
```
Comment (fredguile) [inline src/md-datatable-row.component.scss:6]: Same here, if we could somehow use the $accent color from the theme

**Comment by m-s-sh** [comment]
Just one more note, which I also added like a issue, in new material 2, md-color is mat-color. No matter what it is my favorite datatable!

**Review by fredguile — COMMENTED** [review]
Back in town, I'm going to review it....

File: src/_md-datatable-theme.scss (hunk 1 above)
Comment (fredguile) [reply src/_md-datatable-theme.scss:102]: Just tested it, nevermind, it looks fine like you did

File: src/md-datatable-row.component.scss (hunk 2 above)
Comment (fredguile) [reply src/md-datatable-row.component.scss:6]: Regarding this change, I have to set the selected/hover colors according to the theme's $accent color I'm sorry. I will add a commit on top of yours to have this set.

**Comment by fredguile** [comment]
Except the minor comment I've made regarding the hover/selected color, many thanks for your contribution! I'll merge it and prepare a release with other additions very soon.

**Pull request merged**